packages/core/src/client.ts

```diff
@@ -331,4 +332,8 @@ export class Client {
     const operation = this.createRequestOperation('mutation', query, opts);
     return this.executeRequestOperation(operation);
   };
+
+  get debugSource() {
```

Comment:
Open to omitting this and just using the debugSubject property 🤷

Reply:
I'd just use the property, so we don't even have the compiled getter; the property will be accessible anyway, so shouldn't make a usage difference :)